app/views/applications/show.html.erb

```diff
@@ -0,0 +1,63 @@
+<h1><%= @application.name %></h1>
+
+<h3>Street Address: <%= @application.street_address %></h3>
+<h3>City: <%= @application.city %></h3>
+<h3>State: <%= @application.state %></h3>
+<h3>Zip Code: <%= @application.zip_code %></h3>
+<h3>Reason for applying: <%= @application.description %></h3>
+<h3>Application Status: <%= @application.status %></h3>
+
+<h3>Applying for pets: </h3>
+<ul>
+    <% @application.pets.each do |pet| %>
+        <li><p><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></p></li>
+    <% end %>
+</ul>
+
+<% if @application.pets.present? %>
+    <% if @application.status == "In Progress" %>
+        <%= form_with url: "/applications/#{@application.id}", method: :get, local: true do |f| %>
```

Comment:
As discussed in controller comments, a more RESTful option here would be to route to `PATCH "/applications/:id"` to handle the update of application status, and then that update action could redirect to the show page